src/xcode/ENA/ENA/Source/Client/HTTP Client/HTTPClient.swift

```diff
@@ -403,89 +352,6 @@ final class HTTPClient: Client {
 		})
 	}
 	
-	// swiftlint:disable:next cyclomatic_complexity
-	func getDigitalCovid19Certificate(
-		registrationToken token: String,
-		isFake: Bool,
-		completion: @escaping DigitalCovid19CertificateCompletionHandler
-	) {
-		guard let request = try? URLRequest.dccRequest(
-				configuration: configuration,
-				registrationToken: token,
-				headerValue: isFake ? 1 : 0
-		) else {
-			completion(.failure(.urlCreationFailed))
```

Comment:
This error is not handled in the new code. Did you consider it?

Reply:
Yes, our locators are not failable, so no need to handle this anymore

Reply:
And we have a default `invalidRequestError` which is returned by the service when the URL creation fails.